### PR TITLE
SE-2413 ssh access to jenkins

### PR DESCRIPTION
**Pull request by bkochendorfer**
Adding the Web SRE team to users and removing other users. This will allow us to ssh into the Jenkins server to troubleshoot any issues.

### CODE REVIEW

**Comment by bkochendorfer** [comment]
@mdn/sre Can't tag this team as a reviewer. @escattone could you add that team to this repo as well?

**Comment by escattone** [comment]
@bkochendorfer I just added the @mdn/sre team as maintainers of this repo, thanks!

**Review by duallain — APPROVED** [review]

File: vars/users.yaml [hunk 1]
```diff
@@ -2,8 +2,11 @@
...
 
 account_names:
-  - { user: 'elim', key: 'https://github.com/limed.keys' }
-  - { user: 'jwhitlock', key: 'https://github.com/jwhitlock.keys' }
-  - { user: 'rjohnson', key: 'https://github.com/escattone.keys' }
-  - { user: 'jmize', key: 'https://github.com/jgmize.keys' }
-  - { user: 'dparfitt', key: 'https://github.com/metadave.keys' }
+  # Web SRE Team
+  # According to According to https://github.com/orgs/mozilla-it/teams/it-se/members?query=membership:child-team
+  - { user: 'duallain', key: 'https://github.com/duallain.keys' }
```
Comment (duallain) [inline vars/users.yaml:7]: should we keep escattone's key in here?

jwhitlock still works at moz, but is probably safe to remove.  everyone else should for sure be deleted.

**Review by duallain — APPROVED** [review]

File: vars/users.yaml (hunk 1 above)
Comment (bkochendorfer) [reply vars/users.yaml:7]: sure, good idea!

Comment (jwhitlock) [reply vars/users.yaml:7]: 👍 to removing that @jwhitlock guy

**Pull request merged**